src/simplify_qdq.cpp

```diff
@@ -416,6 +416,28 @@ void remove_qdq_pairs(module& m)
     }
 }
 
+void remove_zero_point(module& m)
+{
+    for(auto ins : iterator_for(m))
+    {
+        if(ins->name() != "dequantizelinear")
```

Comment:
For case of `quantizelinear`, could this not replace a bunch of `0`s with a scalar broadcast..? Thanks.